nativescript-core/Application.md

```diff
@@ -0,0 +1,690 @@
+---
+title: Application
+---
+
+# Application
+
+The Application module provides abstraction over the platform-specific application implementations. The module lets you manage things such as handling lifecycle events of your application (cross-platform and/or native), sending Broadcasts on Android or adding a Notification observer on IOS.
+
+### Import
+
+/// flavor javascript
+
+```javascript
+import { Application } from '@nativescript/core'
+```
+
+///
+
+/// flavor typescript
+
+```typescript
+import { Application } from '@nativescript/core'
+```
+
+///
+
+## Android
+
+For android, you can access the main activity for the application, currently active activity, a method to register a broadcast receiver and other Android-specific properties and methods, as we will see below.
+
+## Android Properties
+
+### android
+
+The property gives you the Nativescript wrapper, the AndroidApplication object, around the native android native application instance.
+/// flavor javascript
+
+```javascript
+const androidApp = Application.android
+```
+
+///
+
+/// flavor typescript
+
+```typescript
+const androidApp: AndroidApplication = Application.android
+```
+
+///
```

Comment:
Removing the `/// flavor X` wrappers here would work, as any subsequent code-blocks will be grouped into tabs, for example (using ' instead of backticks for the example as otherwise it wouldn't render in my comment)

```
'''javascript
 // ...
'''

'''typescript
 // ...
'''
```

will produce a javascript/typescript tab group.